tools/cli/adminTimers.go

```diff
@@ -32,15 +32,13 @@ import (
 
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/backoff"
-	"github.com/uber/cadence/common/log/loggerimpl"
 	"github.com/uber/cadence/common/persistence"
-	"github.com/uber/cadence/common/persistence/cassandra"
-	"github.com/uber/cadence/common/quotas"
 )
 
 // Loader loads timer task information
 type Loader interface {
 	Load() []*persistence.TimerTaskInfo
+	Close()
```

Comment:
Nit: Interface name and comment is now misleading. Ref https://golang.org/doc/effective_go#interface-names